src/App.css

```diff
@@ -1,3 +1,25 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+}
+
 html {
   font-size: calc(15px + 0.390625vw);
 }
+
+.active .nav-link-text {
+  text-decoration: underline;
+}
+
+.nav-item {
+  cursor: pointer;
+  text-decoration: none;
+  margin-right: 20px;
+  font-size: 1rem;
+}
+
+.separator {
+  margin-right: 10px;
+}
```

Comment:
I'd suggest you use react bootstrap for the styling. Everything here can be done with Bootstrap. This is a good opportunity to learn it.